packages/core/src/memory/chat-memory-buffer.ts

```diff
@@ -34,7 +34,7 @@ export class ChatMemoryBuffer<
   }
 
   getMessages(
-    input?: MessageContent | undefined,
+    input?: ChatMessage<AdditionalMessageOptions>[] | undefined,
```

Comment:
this should be `MessageContent` actually, according to python side API

Reply:
should we implement this function in context engine?

https://github.com/run-llama/llama_index/blob/a18b94699ac4e49b17f3f49879adf29dfc7c3ed3/llama-index-core/llama_index/core/chat_engine/context.py#L151